### PR TITLE
Move child menu logic to MonsterListMenu class

### PR DESCRIPTION
**Pull request by RheingoldRiver**

### CODE REVIEW

File: padinfo/menu/id.py [hunk 1]
```diff
@@ -96,13 +96,14 @@ def get_next_monster_id(db_context: "DbContext", monster: "MonsterModel", use_ev
...
         # re-serialize the ims, with the same information in place
-        pane_type = ims.get('pane_type')
+        pane_type = ims.get('pane_type') or IdView.VIEW_TYPE
         pane_type_to_func_map = IdMenuPanes.pane_types()
         response_func = pane_type_to_func_map[pane_type]
         return await response_func(message, ims, **data)
 
     @staticmethod
     async def respond_with_delete(message: Optional[Message], ims, **data):
+        print(ims)
```
Comment (TsubakiBotPad) [inline padinfo/menu/id.py:106]: .

File: padinfo/menu/monster_list.py [hunk 2]
```diff
@@ -6,11 +6,11 @@
 from tsutils import char_to_emoji
 
 from padinfo.menu.common import emoji_buttons, MenuPanes
-from padinfo.menu.id import IdMenu, IdMenuPanes
+from padinfo.menu.id import IdMenu, IdMenuPanes, IdMenuPaneNames
 from padinfo.view.id import IdView
 from padinfo.view.monster_list import MonsterListView, MonsterListViewState
 
-menu_emoji_config = EmbedMenuEmojiConfig(delete_message='\N{CROSS MARK}')
+menu_emoji_config = EmbedMenuEmojiConfig()
```
Comment (TsubakiBotPad) [inline padinfo/menu/monster_list.py:13]: remove altogether?

File: padinfo/menu/monster_list.py [hunk 3]
```diff
@@ -117,25 +108,40 @@ def monster_list_control(state: MonsterListViewState):
...
-        MonsterListMenu.respond_with_2: (char_to_emoji('2'), IdView.VIEW_TYPE),
-        MonsterListMenu.respond_with_3: (char_to_emoji('3'), IdView.VIEW_TYPE),
-        MonsterListMenu.respond_with_4: (char_to_emoji('4'), IdView.VIEW_TYPE),
-        MonsterListMenu.respond_with_5: (char_to_emoji('5'), IdView.VIEW_TYPE),
-        MonsterListMenu.respond_with_6: (char_to_emoji('6'), IdView.VIEW_TYPE),
-        MonsterListMenu.respond_with_7: (char_to_emoji('7'), IdView.VIEW_TYPE),
-        MonsterListMenu.respond_with_8: (char_to_emoji('8'), IdView.VIEW_TYPE),
-        MonsterListMenu.respond_with_9: (char_to_emoji('9'), IdView.VIEW_TYPE),
-        MonsterListMenu.respond_with_10: ('\N{KEYCAP TEN}', IdView.VIEW_TYPE),
+        MonsterListMenu.respond_with_monster_list: (emoji_buttons[MonsterListPaneNames.home], MonsterListPaneNames.home, True, False),
```
Comment (TsubakiBotPad) [inline padinfo/menu/monster_list.py:130]: document what all this true false stuff is

File: padinfo/padinfo.py [hunk 4]
```diff
@@ -223,31 +209,18 @@ async def test_reaction_add(self, reaction, member):
...
-            # It's also better from a perceived performance standpoint becuase the emojis are so rate-limited and
-            # the reset wouldn't happen until all emojis showed up in the child, so this way it feels like everything
-            # happens faster, but regardless the reset must happen first.
-            if respond_with_child[(ims['menu_type'], emoji_clicked)]:
-                await embed_menu.transition(message, ims, emoji_clicked, member, **data)
-            await embed_menu.transition(child_message, next_child_ims, emoji_clicked, member, **data)
-            return
-
-        await embed_menu.transition(message, ims, emoji_clicked, member, **data)
+        if menu_1_ims.get('child_message_id'):
```
Comment (TsubakiBotPad) [inline padinfo/padinfo.py:212]: turn into while loop

File: padinfo/padinfo.py [hunk 5]
```diff
@@ -223,31 +209,18 @@ async def test_reaction_add(self, reaction, member):
...
+            emoji_to_tell_menu2, extra_ims = menu_1_class.get_child_data(menu_1_ims, emoji_clicked)
+            if emoji_to_tell_menu2:
```
Comment (TsubakiBotPad) [inline padinfo/padinfo.py:214]: better var name

File: padinfo/menu/id.py (hunk 1 above)
Comment (RheingoldRiver) [reply padinfo/menu/id.py:106]: oops lol

**Pull request merged**